pkgs/servers/etebase/default.nix

```
@@ -0,0 +1,56 @@
+{ lib, python3, fetchFromGitHub }:
+
+let
+  py = python3.override {
+    packageOverrides = self: super: {
+      django = super.django_3;
+    };
+  };
+in
+  with py.pkgs;
+
+buildPythonPackage  rec {
+  pname = "etebase-server";
+  version = "0.7.0";
+  format = "pyproject";
+
+  src = fetchFromGitHub {
+    owner = "etesync";
+    repo = "server";
+    rev = "v${version}";
+    sha256 = "1r2a7ki9w2h3l6rwqa3fzxjlqfj2lbgfrm8lynjhvcdv02s5abbi";
+  };
+
+  patches = [ ./secret.patch ];
+
+  propagatedBuildInputs = with pythonPackages; [
+    asgiref
+    cffi
+    django
+    django-cors-headers
+    djangorestframework
+    drf-nested-routers
+    msgpack
+    psycopg2
+    pycparser
+    pynacl
+    pytz
+    six
+    sqlparse
+  ];
+
+  installPhase = ''
+    mkdir -p "$out/bin" "$out/lib"
```

```suggestion
    mkdir -p $out/bin $out/lib
```
" are not required here because $out can't have spaces.